resources/assets/coffee/_classes/bar-chart.coffee

```diff
@@ -0,0 +1,115 @@
+###
+# Copyright 2016 ppy Pty. Ltd.
+#
+# This file is part of osu!web. osu!web is distributed with the hope of
+# attracting more community contributions to the core ecosystem of osu!.
+#
+# osu!web is free software: you can redistribute it and/or modify
+# it under the terms of the Affero GNU General Public License version 3
+# as published by the Free Software Foundation.
+#
+# osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+###
+
+class @BarChart
+  transition: 1000
+
+  margins:
+    top: 0
+    right: 0
+    bottom: 0
+    left: 0
+
+  options:
```

Comment:
as I learned today, apparently this shouldn't be done [1].

[1] (scroll down to warning section) https://coffeescript-cookbook.github.io/chapters/classes_and_objects/class-variables-and-instance-variables